packages/mux-player/src/media-theme-mux.html

```diff
@@ -379,9 +379,9 @@
     </template>
 
     <template partial="CaptionsMenuButton">
-      <media-captions-menu-button
+      <media-captions-selectmenu
         default-showing="{{defaultShowingCaptions}}"
-        part="bottom captions menu-button"
```

Comment:
Updated from `menu-button` to select menu. I also added `button`, but should I remove it?

Reply:
I'd say no, it shouldn't have a `button` part, but can we export the underlying media-captions-button as its own part in mux player? To channel prior convos we had w/@heff, would be good for folks to not need to know implementation details of the selectmenu

Reply:
If we switch from captions-selectmenu to chrome-selectmenu and slot the captions-button manually, we could export it as a `button`

Reply:
ah right it's in the light DOM now. We could also do it programmatically (since it's on the light DOM wrt `media-captions-selectmenu`) but that feels a bit too scope creepy for this iteration.